rhel/SPECS/pmm-update.spec

```diff
@@ -1,3 +1,6 @@
+%undefine _missing_build_ids_terminate_build
+%define _binaries_in_noarch_packages_terminate_build   0
```

Comment:
Why not make it a normal x86-64 package? Why keep it nonarch and add this hack?

Reply:
it is ok to use it. Using it we will avoid the following issue in case  pmm-update rpm is updated

https://pmm.cd.percona.com/job/pmm-submodules/job/PR-348/8/console
```
\"msg\": \"Error:  Multilib version problems found. This often means that the root
....
   Protected multilib versions: pmm-update-2.0.0-7.beta5.1907230717.f815fbd.el7.noarch != 1:pmm-update-2.0.0-9.beta5.1907232020.f815fbd.el7.x86_64\\n\", ", 
```

Reply:
I have added comment